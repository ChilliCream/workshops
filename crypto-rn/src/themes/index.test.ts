import * as Themes from './index';

describe('Themes', () => {
  it('should keep shape', () => {
    expect(Themes).toMatchSnapshot();
  });
});
