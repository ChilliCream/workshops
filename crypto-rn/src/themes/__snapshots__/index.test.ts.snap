// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Themes should keep shape 1`] = `
{
  "defaultTheme": {
    "pallete": {
      "brand": {
        "primary": "#D82034",
      },
      "foreground": "#000000",
    },
    "typography": {
      "title": {
        "fontSize": 28,
        "letterSpacing": 0.56,
        "lineHeight": 36,
      },
    },
  },
}
`;
